### DISCUSSION TITLE
Открытие внутренней таблицы при добавлении строки в цикловую.

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#139] [Как это сделать?] [answered]
При добавлении строки в цикловую, открывается внутренняя таблица...  Как сделать чтоб я мог добавлять и внутрь не проваливаться.

**Comment by erpic-ru** [comment]
Теоретически, можно сделать отдельную простую таблицу, которая будет кодами создавать и добавлять значения в расчетную в цикле со ссылкой на переход в цикл. Но зачем?

**Comment by ttm-alexeypolunin** [comment]
Сегодня будет видео по этому вопросу

**Comment by ttm-alexeypolunin** [comment, marked as answer]
### [Видео на YouTube](https://youtu.be/950C-q44REM)
